models/articles.models.js

```diff
@@ -27,6 +30,35 @@ exports.fetchAllArticles = () => {
   });
 };
 
+exports.fetchArticles = (topic) => {
+  if (!topic)
+    return Promise.reject({ status: 400, errorMessage: invalidInput });
```

Comment:
We will need to think about how we can ensure, using one model that we check that the topic exists before allowing that value to be processed through our database query and therefore exposing ourselves to SQL injection. Have a think about how we can utilise some kind of check on the topics table before calling the model, if we have a topic query.